### ISSUE TITLE
How to create/use plugin 

### ISSUE THREAD

**Issue by jbmyid** [babel/ruby-babel-transpiler#298]
I have a js code for Babel plugin which can be used as a plugin in js Babel transform, how do I use the same plugin in here?

**Comment by oskarkrawczyk** [comment]
@jbmyid I don't think you can, in theory the gem should take any of the options that Babel transform does, but in practice it only seems to understand `ast`, `code`, not much more.

I doesn't looks like the Gem is maintained anymore.
